### ISSUE TITLE
error while converting to tflite model

### ISSUE THREAD

**Issue by pritammahadik94** [tensorflow/models#6411]
Please go to Stack Overflow for help and support:

### System information
- **What is the top-level directory of the model you are using**:
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: 18.0
- **CUDA/cuDNN version** :none
- **GPU model and memory**: none 
- **Exact command to reproduce**:
bazel run --config=opt tensorflow/contrib/lite/toco:toco -- --input_file=temp/box.pb --output_file=temp/detect.tflite -input_shapes=1,300,300,3--input_arrays=normalized_input_image_tensor --output_arrays='TFLite_Detection_PostProcess','TFLite_Detection_PostProcess:1','TFLite_Detection_PostProcess:2','TFLite_Detection_PostProcess:3' --inference_type=QUANTIZED_UINT8 --mean_values=128 --std_values=128 --change_concat_input_ranges=false --allow_custom_ops

### Describe the problem
We trained the custom object detection model for the box(inventory box) detection using tensorflow object detection API. We used weights of pretrained  ssd_mobilenet_v1_coco models for training. 
Then we convert frozen graph for the model from checkpoints using export_tflite_ssd_graph.py file.
Now I wanted to convert an optimized model by using TOCO so I can run our model on android devices. 
We followed all the mentioned steps, also we tried with various version of bazel (0.22.0, 0.20.0, 0.19.1) but not able to transform the model successfully.
While running above command with bazel 0.18.0 I got the following error. Please help me with the same.
Also, is there any update to convert a frozen inference graph of Faster RCNN model to Tflite, as iI read from Tensorflow API documentation, currently SSD mobilenet model only support for the tflite conversion.

### Source code / logs
ERROR: D:/tensorflow/tensorflow/contrib/lite/BUILD:34:1: C++ compilation of rule '//tensorflow/contrib/lite:arena_planner' failed (Exit 2): cl.exe failed: error executing command
  cd C:/users/pritamm1/_bazel_pritamm1/26orbg4z/execroot/org_tensorflow
  SET INCLUDE=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt;
    SET PATH=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64;C:\windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE;C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Windows Kits\10\bin\x86;;C:\windows\system32
    SET PWD=/proc/self/cwd
    SET PYTHON_BIN_PATH=C:/Users/pritamm1/AppData/Local/conda/conda/envs/tensorflow111/python.exe
    SET PYTHON_LIB_PATH=C:/Users/pritamm1/AppData/Local/conda/conda/envs/tensorflow111/lib/site-packages
    SET TEMP=C:\Users\pritamm1\AppData\Local\Temp
    SET TF_DOWNLOAD_CLANG=0
    SET TF_NEED_CUDA=0
    SET TF_NEED_OPENCL_SYCL=0
    SET TMP=C:\Users\pritamm1\AppData\Local\Temp
  C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64/cl.exe /nologo /DCOMPILER_MSVC /DNOMINMAX /D_WIN32_WINNT=0x0600 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_SECURE_NO_WARNINGS /bigobj /Zm500 /EHsc /wd4351 /wd4291 /wd4250 /wd4996 /I. /Ibazel-out/x64_windows-opt/genfiles /Ibazel-out/x64_windows-opt/bin /Iexternal/bazel_tools /Ibazel-out/x64_windows-opt/genfiles/external/bazel_tools /Ibazel-out/x64_windows-opt/bin/external/bazel_tools /showIncludes /MD /O2 /Oy- /DNDEBUG /wd4117 -D__DATE__="redacted" -D__TIMESTAMP__="redacted" -D__TIME__="redacted" /Gy /Gw -w /arch:AVX /Fobazel-out/x64_windows-opt/bin/tensorflow/contrib/lite/_objs/arena_planner/arena_planner.obj /c tensorflow/contrib/lite/arena_planner.cc
.\tensorflow/contrib/lite/context.h(183): error C2144: syntax error: 'float' should be preceded by ';'
.\tensorflow/contrib/lite/context.h(183): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
Target //tensorflow/contrib/lite/toco:toco failed to build
INFO: Elapsed time: 33.305s, Critical Path: 1.15s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
